src/chainparams.h

```diff
@@ -146,6 +146,32 @@ class CChainParams
     ChainTxData chainTxData;
 };
 
+/**
+ * SigNetOptions holds configurations for creating a signet CChainParams.
+ */
+struct SigNetOptions {
+    const std::vector<uint8_t> challenge;
+    std::vector<std::string> seeds;
+};
+
+/**
+ * Return SigNetOptions filled with default values.
```

Comment:
I think it's not really "SigNetOptions filled with default values" but rather "Options for the default signet". I remember @theuni made a good point offline that it almost never makes sense to override _some_ of the options for the default signet.